zuul.d/jobs.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- project:
+    check:
+      jobs:
+        - manila-operator-kuttl
+- job:
+    name: manila-operator-kuttl
+    parent: cifmw-kuttl
+    attempts: 1
+    vars:
+      cifmw_kuttl_tests_operator_list:
+        - manila
+    required-projects:
```

Comment:
those projects are already defined in parent job, so you may not need to define them here again :)

Reply:
+1, removing in my next PS!